src/org/labkey/test/components/react/MultiMenu.java

```diff
@@ -186,34 +206,182 @@ public List<String> getMenuText()
+    {
+        var itemTexts = getWrapper().getTexts(getItemsUnderHeading(heading));
+        if (collapse)
+            collapse();
+        return itemTexts;
+    }
+
+    /**
+     * gets the dropdown-section__menu-items between a heading and the next separator
+     *
+     * @param heading
+     * @return
+     */
+    protected List<WebElement> getItemsUnderHeading(String heading)
     {
         expandAll();
         boolean headingFound = false;
-        List<String> items = new ArrayList<>();
+        List<WebElement> itemsUnderHeading = new ArrayList<>();
         List<WebElement> listItems = Locator.tag("li").findElements(this);
 
         for (WebElement item : listItems)
         {
             String className = item.getAttribute("class");
+            String role = item.getAttribute("role");
             String text = item.getText().trim();
 
             if (className.contains("dropdown-header") && text.equalsIgnoreCase(heading))
                 headingFound = true;
 
-            // Once we've found our header we know that all dropdown-section__menu-item elements belong to the heading
+            // Once we've found our header we know that all presentation elements belong to the heading
             // we are interested in
-            if (headingFound && className.contains("dropdown-section__menu-item"))
-                items.add(text);
+            if (headingFound && role.equals("presentation"))
+                itemsUnderHeading.add(item);
 
             // Once we hit a divider we're done looking at menu items related to the heading, so we can stop iterating
-            if (headingFound && className.contains("divider"))
+            if (headingFound && role.equals("separator"))
                 break;
         }
 
-        collapse();
+        return itemsUnderHeading;
+    }
+
+    private WebElement getItemUnderHeading(String toggleLabel, String label)
+    {
+        var itemsUnderHeading = getItemsUnderHeading(toggleLabel);
+        for (WebElement item : itemsUnderHeading)
+        {
+            if (item.getText().trim().equals(label))
+                return item;
+        }
+        throw new NoSuchElementException("No item '" + label + "' under toggle '" + toggleLabel + "' found.");
+    }
 
-        return items;
+    /**
+     * Clicks the specified item under the specified heading
+     *
+     * @param heading heading under which to find the item
+     * @param label   text of the item
+     */
+    public void clickItemUnderHeading(String heading, String label)
+    {
+        WebElement item = getItemUnderHeading(heading, label);
+        getWrapper().shortWait().until(ExpectedConditions.elementToBeClickable(item));
+        item.click();
+    }
+
+    /**
+     * gets the names of dropdown-section__menu-items between a heading and the next divider
+     *
+     * @param heading The text of the heading
+     * @param collapse Whether to collapse the menu when finished
+     * @return
```

Comment:
`heading` ≠ `toggle`
```suggestion
     * gets the names of dropdown-section__menu-items between a toggle and the next toggle or divider
     *
     * @param heading The text of the toggle
     * @param collapse Whether to collapse the menu when finished
     * @return texts of menu items under toggle
```